arrow/memory/memory_test.go

```diff
@@ -123,3 +124,11 @@ func BenchmarkSet_8000(b *testing.B) {
 func BenchmarkSet_8192(b *testing.B) {
 	benchmarkSet(b, 8192)
 }
+
+func TestSetNoClobberFramePointer(t *testing.T) {
+	var b [8]byte
+	memory.Set(b[:], 0xff)
+	// We should be able to safely frame pointer unwind after calling this
+	// function
+	tools.FPUnwind()
```

Comment:
let's add a comment referencing the issue so that we understand why this test exists in the future

Reply:
Done 👍